### PR TITLE
fixed #23

### LINKED ISSUE #23: missing session requirements in high-level workflows
The high-level workflows in `current-development-phase.md` are inconsistent in their inclusion of session requirements. Edit for consistency and completeness.

### PR DESCRIPTION
**Pull request by indomitableSwan**
Fixes #23. In adding session details, realized some additional clarity was required around local actions that don't require a session with the server, so adjusted wording.

### CODE REVIEW

File: current-development-phase.md [hunk 1]
```diff
@@ -22,16 +22,22 @@ We provide sketches of the generation, storage, and use of arbitrary secrets bel
...
+    1. Retrieval from a key server MUST occur via a mutually authenticated channel that satisfies confidentiality and integrity. 
+    1. The key retrieved MUST be a key associated to the authenticated user.
+    1. A retrieved secret can be:
+        1. Stored locally in secure storage on the user's device.
+        1. Copied to the system clipboard.
+1. The asset owner may _import_ a secret to the system in an appropriate format. 
     1. The default expected format is as bytes of the form ``len || secret``, where `len` is 1 byte that represents the length of the secret `secret`.
     1. The implementation may define other acceptable import formats.
-1. The asset owner may _export_ the secret from the key server. The export format for the key SHOULD allow for easy transfer of the key material to another digital asset management system, i.e., secrets should be portable.
+    1. The user may choose to store the imported secret locally or on the key server.
```
Comment (marsella) [inline current-development-phase.md:34]: Do you want to add a comment here along the lines of, "if it's stored on the key server, export requires a mutually authenticated channel that satisfies confidentiality and integrity"?

File: current-development-phase.md [hunk 2]
```diff
@@ -22,16 +22,22 @@ We provide sketches of the generation, storage, and use of arbitrary secrets bel
...
+1. The asset owner may _export_ the secret from the system. The export format for the key SHOULD allow for easy transfer of the key material to another digital asset management system, i.e., secrets should be portable.
```
Comment (marsella) [inline current-development-phase.md:35]: Ditto above: Consider adding a caveat that if it's exported from the server, it needs a secure channel.

File: current-development-phase.md (hunk 1 above)
Comment (marsella) [reply current-development-phase.md:34]: Or if you want to reduce verbiage throughout, you could probably say "authenticated session" a' la point 2.

**Review by marsella — COMMENTED** [review]
Do you want to add any guarantees about the registration communication session? Does that channel need to be confidential?

File: current-development-phase.md (hunk 1 above)
Comment (indomitableSwan) [reply current-development-phase.md:34]: Yes, I should add that.

File: current-development-phase.md (hunk 2 above)
Comment (marsella) [reply current-development-phase.md:36]: The other comment was for import. This one is for export. You addressed export, but not import. See the line that says
> The user may choose to store the imported secret locally or on the key server.

Although I guess you're referencing "store" which already clarifies the channel requirements.

Comment (indomitableSwan) [reply current-development-phase.md:36]: I'm fine with redundancy.

**Review by marsella — APPROVED** [review]
Looks good, thanks for the fixes!

**Pull request merged**